Makefile

```diff
@@ -8,7 +8,7 @@ endif
 FAIL_ON_STDOUT := awk '{ print } END { if (NR > 0) { exit 1 } }'
 
 CURDIR := $(shell pwd)
-path_to_add := $(addsuffix /bin,$(subst :,/bin:,$(GOPATH)))
+path_to_add := $(addsuffix /bin,$(subst :,/bin:,$(GOPATH))):$(PWD)/tools/bin
```

Comment:
I think `$(PWD)/tools/bin` is ugly.
Any way to achieve elegantly？

Reply:
$(CURDIR)/tools/bin